### PR TITLE
26685 26687 view progress timelines

### PR DESCRIPTION
**Pull request by dletorey**
### Description

Added the experimental release not for `view-timeline-name` CSS Property and `view()` CSS Function.
### Motivation

Completing [26687](https://github.com/mdn/content/issues/26687) and [26685](https://github.com/mdn/content/issues/26685) issues.

### Related issues and pull requests

Related [Content PR - 27075](https://github.com/mdn/content/pull/27075) by @chrisdavidmills 
Related [BCD PR 20132](https://github.com/mdn/browser-compat-data/pull/20132)

### CODE REVIEW

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 1]
```diff
@@ -881,6 +881,88 @@ See ([Firefox bug 1794628](https://bugzil.la/1794628)) for more details.
   </tbody>
 </table>
 
+### Named view progress timelines property
+
+The CSS [`view-timeline-name`](/en-US/docs/Web/CSS/view-timeline-name) property lets you give a name to a `view-timeline`, that is referenced with a [`<custom-dent>`](/en-US/docs/Web/CSS/custom-ident), to animate elements as they appear in the scroller.
```
Comment (hamishwillee) [inline files/en-us/mozilla/firefox/experimental_features/index.md:886]: The way I read the animation is not so much "as they appear", as "as they move through". How about something like ...

```suggestion
The CSS [`view-timeline-name`](/en-US/docs/Web/CSS/view-timeline-name) property lets you give a name to a `view-timeline`. A named view progress timeline is used to animate the associated element as it moves through the visible area of its ancestor scroller.
```

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 2]
```diff
@@ -881,6 +881,88 @@ See ([Firefox bug 1794628](https://bugzil.la/1794628)) for more details.
...
+    <tr>
+      <th>Preference name</th>
+      <td colspan="2"><code>layout.css.scroll-driven-animations.enabled</code></td>
+    </tr>
+  </tbody>
+</table>
+
+### Anonymous view progress timelines function
+
+The CSS [`view()`](/en-US/docs/Web/CSS/animation-timeline/view) function lets you anonymously animate elements as they appear in the scroller.
```
Comment (hamishwillee) [inline files/en-us/mozilla/firefox/experimental_features/index.md:927]: Never sure how much time to spend on these things. How about this though.

```suggestion
The CSS [`view()`](/en-US/docs/Web/CSS/animation-timeline/view) function lets you specify that the `animation-timeline` for an element is a view progress timeline, which will animate the element as it moves through the visible area of its ancestor scroller.
The function defines the axis of the parent element that supplies the timeline, along with the inset within the visible area at which the animation starts and begins.
```

**Comment by hamishwillee** [comment]
@dletorey Sorry, I missed that this review had fallen on me. If you don't get a review in the first day I work after you, essentially Monday or Friday, then feel free to ping me on slack. It means I missed the notification.

**Comment by hamishwillee** [comment]
@dletorey Is this waiting on me to look at again?

**Comment by dletorey** [comment]
> @dletorey Is this waiting on me to look at again?

Yes please

**Review by hamishwillee — APPROVED** [review]
Looks very good to me.

**Pull request merged**